google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -75,11 +96,11 @@ class InstanceMetadata:
     def __init__(
         self,
         ephemeral_cert: str,
-        ip_address: str,
+        ip_addrs: Dict[str, Any],
```

Comment:
In the new version of the Go connector, I've made the default "public" and required them to opt into private to help make it more of an explicit choice. What do you think about doing the same here?

https://github.com/kurtisvg/cloud-sql-go-connector/blob/925d6c2686d519d182dc196c752ed0c7edb0e28c/dialer.go#L93

Reply:
I agree, let's keep them consistent. We should probably also make the same update in the Java connector: https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/5ceacadbca5d91018b53a8131f3b5b110d70155d/core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java#L78

Reply:
This is probably a breaking change in the Java connector, but I think we should introduce it whenever we decide to next do a major version bump